### ISSUE TITLE
Compiler warnings on Xcode 8.3

### ISSUE THREAD

**Issue by matti777** [DaveWoodCom/XCGLogger#192]
```
    - WARN  | xcodebuild:  XCGLogger/Sources/XCGLogger/Filters/UserInfoFilter.swift:71:29: warning: @discardableResult declared on a function returning Void is unnecessary
    - WARN  | xcodebuild:  XCGLogger/Sources/XCGLogger/Filters/FileNameFilter.swift:71:29: warning: @discardableResult declared on a function returning Void is unnecessary
```

With XCGLogger 4.0.0 from Cocoapods.

**Comment by vrutberg** [comment]
I addressed this in a PR I just opened: #197

**Comment by DaveWoodCom** [comment]
This has been resolved in the new 5.0.1 version. Please let me know if you see any other issues.
